scripts/run_variant_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd lib/variant
+mkdir -p build
+cd build
+cmake -DMPARK_VARIANT_INCLUDE_TESTS="mpark" -DCMAKE_CXX_FLAGS="-std=c++11" ..
```

Comment:
Are we sure that `cmake` is installed prior to running this (as this is being run in all Travis jobs)?

Reply:
Hmm good point - what would you suggest - the project is in CMake not Make? (one option of course being: abandon this approach!)

Reply:
I don't think there's a problem asking for `cmake` to be installed. Just make sure it's documented/actually done in CI configurations.